src/domain/swaps/entities/__tests__/order.builder.ts

```diff
@@ -0,0 +1,101 @@
+import { Order } from '@/domain/swaps/entities/order.entity';
+import { Builder, IBuilder } from '@/__tests__/builder';
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+
+export function orderBuilder(): IBuilder<Order> {
+  return new Builder<Order>()
+    .with('sellToken', getAddress(faker.finance.ethereumAddress()))
+    .with('buyToken', getAddress(faker.finance.ethereumAddress()))
+    .with(
+      'receiver',
+      faker.datatype.boolean()
+        ? getAddress(faker.finance.ethereumAddress())
+        : null,
+    )
+    .with('sellAmount', faker.number.bigInt({ min: 1 }))
+    .with('buyAmount', faker.number.bigInt({ min: 1 }))
+    .with('validTo', faker.date.future().getTime())
+    .with('appData', faker.string.hexadecimal())
+    .with('feeAmount', faker.number.bigInt({ min: 1 }))
+    .with('kind', faker.helpers.arrayElement(['buy', 'sell']))
```

Comment:
The main reason why I didn't add `unknown` here it's because the default builder should return a "sensible" default i.e.: in a testing scenario, you should most of the times expect a "normal" Order.

However this only applies to the default values. Setting `unknown` is of course still allowed in a testing scenario. We would otherwise need to write the expected values most of the time to avoid having `unknown`.

Reply:
Ok, it makes sense, thanks for the explanation 🙂